ci/build.sh

```diff
@@ -106,7 +106,9 @@ bundle_fluentd_plugins "${VERSION}" || (echo "Failed bundling fluentd plugins" &
 
 echo "Building docker image with $DOCKER_TAG:local in $(pwd)..."
 cd ./deploy/docker || exit 1
-docker build . -f ./Dockerfile -t "$DOCKER_TAG:local" --no-cache
+readonly no_cache=$([ "$DOCKER_USE_CACHE" == "true" ] && echo "" || echo "--no-cache")
+# shellcheck disable=SC2086
+docker build . -f ./Dockerfile -t "$DOCKER_TAG:local" ${no_cache}
```

Comment:
per https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions
```suggestion
docker build . -f ./Dockerfile -t "$DOCKER_TAG:local" ${no_cache:+"no_cache"}
```

Reply:
@perk-sumo please see the change